src/main/java/com/example/mate/exception/EntityNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package com.example.mate.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
```

Comment:
Interesting! I didn't know this works)
Don't forget to remove it when you add a global exception handler.